sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -59,6 +59,10 @@ public partial class ServiceBusClient : System.IAsyncDisposable
     {
         protected ServiceBusClient() { }
         public ServiceBusClient(string connectionString) { }
+        public ServiceBusClient(string fullyQualifiedNamespace, Azure.AzureNamedKeyCredential credential) { }
+        public ServiceBusClient(string fullyQualifiedNamespace, Azure.AzureNamedKeyCredential credential, Azure.Messaging.ServiceBus.ServiceBusClientOptions options) { }
+        public ServiceBusClient(string fullyQualifiedNamespace, Azure.AzureSasCredential credential) { }
+        public ServiceBusClient(string fullyQualifiedNamespace, Azure.AzureSasCredential credential, Azure.Messaging.ServiceBus.ServiceBusClientOptions options) { }
```

Comment:
Same comment here - I think we want to make options a default parameter.

Reply:
I'm good with that, and it's what we do elsewhere;  I didn't change the pattern that was already in place.     If we're doing these, I think that we want to do the same for the `TokenCredential` overload as well, no?